index.js

```diff
@@ -39,7 +39,7 @@ export async function acs (pubSubMessage, context) {
     const csvData = new Audience(pubSubMessage.data).acsCsv
     await s3.upload({
       Bucket: process.env.S3_BUCKET,
-      Key: `tealium_to_acs_${new Date().toISOString()}.csv`,
+      Key: `tealium_to_acs_${new Date().toISOString().slice(0,10)}.csv`,
```

Comment:
The file name ```tealium_to_acs_2021-05-12T19:52:20.111Z.csv``` throws an error in ACS when trying to transfer the file. So John requested we just use the date, minus the time. So now the file name is ```tealium_to_acs_2021-05-12.csv```.